js/scholarships.js

```diff
@@ -92,6 +98,9 @@
 		$('.wsuwp-scholarships-form').find('option:selected').removeAttr('selected');
 		$('.wsuwp-scholarships-form').find('input:not(:submit)').val('');
 
+		// Clear session storage.
+		sessionStorage.clear();
```

Comment:
It might be better to use `sessionStorage.removeItem()` here, especially if combined with the suggestion to use one object to store all of the data. Otherwise (I might be getting this wrong), the script could be stomping on any other session storage added by other plugins. Unlikely that it's happening, but you never know. :)